### DISCUSSION TITLE
How to add circuit breaker?

### DISCUSSION THREAD

**Discussion by hruslan2000** [make-all/tuya-local#699] [Q&A]
Help me, please!
In new version added support for WDYK 3 Phase 4 pole 400V circuit breaker with Energy Monitoring (https://github.com/make-all/tuya-local/issues/550).
I have this one, but I dont understand how to add it. 

When I add device, local tuya not see DPs 101 (Voltage), 102 (Current), 103 (Active Power). If I add DPs manual -  show "-1". 
When I see data with tuya iot - it show like this "101": "CQEJBglW" (three phase). I dont know how to do.
Maybe anyone help me...
PS. Sorry for my bad English.

**Comment by make-all** [comment]
This Discussion area is not for localtuya configuration discussion.  If you are using this integration, there is no manual DPs configuration.
